pytroll_collectors/segments.py

```diff
@@ -75,6 +75,39 @@ def __init__(self, config):
         self.logger = logging.getLogger("segment_gatherer")
         self._loop = False
 
+        # Convert check time into int minutes variables
+        for key in self._patterns:
+            if "hour_pattern" in self._patterns[key]:
```

Comment:
from what I understand, this patch operates on the start time of the data only, right ? so could this be renamed to reflect this, for example `start_time_pattern` ?

Reply:
Maybe if we call the filter "slot time" is it more generic for all satellites ?

Reply:
"Slot time" is more generic indeed, but then it's the same problem as "hour_pattern", which is that we don't know from the name if it operates on `start_time`, `nominal_time`, `processing_time` or `end_time` for example...

Reply:
For the hour_pattern, really in the code we test the metadata start_time received from the trollstalker, start_time_pattern is the most clear value to be used to compare.